docs/06_how-to-guides/40_multi-index/how-to-define-a-singleton.md

```diff
@@ -84,9 +118,10 @@ class [[eosio::contract]] singleton_example : public contract {
 };
 ```
 
-And below is a possible implementation for the two `get` and `set` actions defined above. It also demonstrates the usage of a couple of singleton methods. Note that the `set` action makes use of the singleton's `set` method, for which parameter is the account to pay for the new value stored. In this case, the same account name that is stored in the primary value is the payer. However, it can be a different account if so required.
+Find below a possible implementation for the two `get` and `set` actions. It also demonstrates the usage of the `get` and `set` singleton methods. Note that the `set` action makes use of the singleton's `set` method, for which the second parameter is the payer account for the RAM needed to store the new value.
```

Comment:
Find below a possible implementation for the two `get` and `set` actions.

how about =>

Below is an example for the `get` and `set` actions.

Reply:
or 

Below is example code  for the get and set actions.

Reply:
or 

The following is ...